### PR TITLE
Added Name

### PR DESCRIPTION
**Pull request by Ashray707**

### CODE REVIEW

**Review by brandondombrowsky — APPROVED** [review]
LGTM

**Review by brandondombrowsky — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@
 - Lucas Knezevich
 - Jason Beutler
 - Victor Navarro
-- Ashray Thapa
```
Comment (alyoshenka) [inline README.md:8]: Not sure if we need a PR since it looks like your name is already in there? But could do another one for more practice

**Review by alyoshenka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alyoshenka) [reply README.md:8]: Ahh I might have read something wrong. Anyways, you can use the web editor to solve the merge conflict, and then it should be good to go.

**Review by LucasKnezevich — APPROVED** [review]

**Review by brandondombrowsky — APPROVED** [review]
This might work now?

File: README.md (hunk 1 above)
Comment (brandondombrowsky) [reply README.md]: Thanks that's good to know. But these looked like: 
NAMBOB>>>>>
=========
LASTOMAS>>>>

Have you seen that before?

Comment (brandondombrowsky) [reply README.md]: Not a lit like above. Just an equal number of dashes on each side.

**Pull request merged**